gordo_components/builder/local_build.py

```diff
@@ -10,9 +10,12 @@
     get_dict_from_yaml,
 )
 from gordo_components.builder.build_model import build_model
+from gordo_components.builder.azure_utils import get_mlflow_logger
 
 
-def local_build(config_str: str) -> Iterable[Tuple[Union[BaseEstimator, None], dict]]:
+def local_build(
+    config_str: str, enable_remote_logging: bool = True, workspace_kwargs: dict = {}
```

Comment:
Should it not be default = False?

Reply:
This should really not have been called that. For the local builder, I instead wanted it to be default log locally to Mlflow, but one could disable Mlflow entirely. While in the CLI, disabling MLflow is just allowing it to log locally rather than log remotely. I've renamed and updated things to make that clearer.